prebid-mobile/modules/rendering/android-sdk-integration-gam.md

```diff
@@ -83,7 +72,7 @@ Pay attention that the `loadAd()` should be called on the main thread.
 {% endcapture %}
 {% include /alerts/alert_warning.html content=warning_note %}
 
-#### Step 1: Create Event Handler
+##### Step 1: Create Event Handler
 {:.no_toc}
 
 GAM's event handlers are special containers that wrap GAM Ad Views and help to manage collaboration between GAM and Prebid views.
```

Comment:
I'm not a native speaker, and this is a legacy sentence, but I think it can be misinterpreted.

GAM Event handlers do not belong to GAM; they are part of Prebid. So maybe `'s` should be removed.

Reply:
```suggestion
Prebid SDK's GAM event handlers are special containers that wrap GAM Ad Views and help to manage collaboration between GAM and Prebid views.
```